pywbem/config.py

```diff
@@ -60,3 +61,11 @@
 #: operations.
 
 DEFAULT_ITER_MAXOBJECTCOUNT = 1000
+
+#: Maximum log entry size. An integer that sets the maximum size of each log
+#: entry if the log_detail_level flag is set.
+
+DEFAULT_MAX_LOG_ENTRY_SIZE = 1000
+
+#: Set default log destination.  None means that there will be no logging
+DEFAULT_LOG_DESTINATION = 'file'
```

Comment:
Sloppy coding. It should all be here.  I believe that the DEFAULT_LOG_SIZE should also be in logging and not config so moved it.